cypress/integration/page.spec.js

```diff
@@ -225,7 +254,7 @@ describe('', () => {
         cy.get('@users').should('have.callCount', 1);
       });
 
-      it('should not request posts from API', () => {
+      it.skip('should not request posts from API', () => {
```

Comment:
This test is skipped. Ensure that it is intentionally skipped and consider enabling it if it is relevant to the test suite.